test/run_tests.sh

```diff
@@ -60,6 +60,14 @@ function run_dynamic {
   fi
 }
 
+function run_eager_debug {
+  echo "Running in Eager Debug mode: $@"
+  IR_DEBUG=ir_debug_file
+  XLA_USE_EAGER_DEBUG_MODE=1 XLA_IR_DEBUG=1 XLA_SAVE_TENSORS_FILE=$IR_DEBUG run_test "$@"
```

Comment:
let's remove `XLA_IR_DEBUG=1 XLA_SAVE_TENSORS_FILE=$IR_DEBUG`, we can skip the `test_pow_constant` in eager debug mode